lisa/runners/lisa_runner.py

```diff
@@ -359,7 +359,10 @@ def _run_test_task(
                 environment.nodes.check_kernel_panics()
             except LisaException as identifier:
                 # not throw exception here, since it will cancel all tasks
-                # just print log here
+                # just print log here and set test result status as failed
+                if "found panic in serial log" in str(identifier):
```

Comment:
Define an exception type like `KernelPanicException`, instead of check by the string.

Reply:
added KernelPanicException